### PR TITLE
Sdt 644 publish design system

### PR DESCRIPTION
**Pull request by rpowis**
This PR updates the markup and some styles to get the Design System looking like the GOV.UK one.

image

image

### CODE REVIEW

File: public/assets/stylesheets/design-system/modules/_technical-documentation.scss [hunk 1]
```diff
@@ -77,7 +78,7 @@
   // Expand margin when h1 followed directly by paragraph
   h1 + p {
     @include media(tablet) {
-      margin-top: $gutter * 1.5;
+      // margin-top: $gutter * 1.5;
```
Comment (csabascript) [inline public/assets/stylesheets/design-system/modules/_technical-documentation.scss:81]: Do you want to keep this?

File: public/assets/stylesheets/design-system/_custom.scss [hunk 2]
```diff
@@ -4,6 +4,28 @@ main {
...
+
+// Full width home page
+
+// Centred
+.column-constrained {
+  padding: 0 $gutter-half;
+  @media (min-width: 1200px ) {
+    padding: 0 10%;
+  }
+  @media (min-width: 1367px ) {
```
Comment (csabascript) [inline public/assets/stylesheets/design-system/_custom.scss:24]: Why these breakpoint values?

File: public/assets/stylesheets/design-system/modules/_masthead.scss [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+ *
+ * <div class="masthead">
+ *  [...]
+ * </div>
+ */
+
+.masthead {
+  background-color: $govuk-blue;
+  padding: $gutter;
+  color: white;
```
Comment (csabascript) [inline public/assets/stylesheets/design-system/modules/_masthead.scss:17]: I would use the variable here.

File: public/assets/stylesheets/design-system/modules/_masthead.scss [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+  width: 101%;
+  margin-left: -32px;
+
+  @include media(mobile) {
+    padding: $gutter-half;
+  }
+
+  a {
+    color: white;
```
Comment (csabascript) [inline public/assets/stylesheets/design-system/modules/_masthead.scss:26]: I would use the colour variable.

**Comment by rpowis** [comment]
@csabascript Anything in the `design-system/modules` stylesheets has come over form the GOV.UK versions 1:1. I'm not going to make any changes to them in order to keep identical as possible. That _should_ make updating easier. At least until they actually publish the template.

**Review by csabascript — APPROVED** [review]

**Pull request merged**